src/App.tsx

```diff
@@ -1,61 +1,134 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList, TodoType, UserType } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+interface Errors {
+  title: string;
+  user: string;
+}
 
 export const App = () => {
+  const [title, setTitle] = useState<string>('');
+  const [author, setAuthor] = useState<string>('0');
+  const [errors, setErrors] = useState<Errors>({ title: '', user: '' });
+
+  const getTodos = (): TodoType[] => {
+    const todos = todosFromServer.map(item => {
+      return {
+        ...item,
+        user: usersFromServer.find(us => us.id === item.userId) as UserType,
```

Comment:
```suggestion
        user: usersFromServer.find(us => us.id === item.userId) || null,
```
try to avoid assertions as much as you can